src/pytestqt/qml/qmlbot.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from pytestqt.qt_compat import qt_api
+
+
+class QmlBot:
+    def __init__(self):
+        self.engine = qt_api.QtQml.QQmlApplicationEngine()
+        main = Path(__file__).parent / "botloader.qml"
+        self.engine.load(main.resolve(True))
+
+    @property
+    def _loader(self):
+        self.root = self.engine.rootObjects()[0]
```

Comment:
Just to confirm: do we want `root`, `engine` and `comp` to be part of the public API?

If so I suggest we change `comp` to `component`, no need to abbreviate it.

Reply:
`comp` not, engine yes, I will rename

Reply:
I see `root` is still public, just in case you missed it.

Reply:
shouldnt be :worried: sorry